supplementary_style_guide/glossary_terms_conventions/general_conventions/u.adoc

```diff
@@ -1,3 +1,14 @@
+[discrete]
+[[udp]]
+==== image:images/yes.png[yes] UDP (noun)
+*Description*: The User Datagram Protocol (UDP) is a connection-less protocol. Clients can send datagrams to servers with a service that is listening for UDP traffic without establishing a connection. UDP prefers time and overhead over reliability.
```

Comment:
Does the last sentence mean that UDP prefers _minimizing_ time and overhead over _ensuring_ reliability?

Reply:
Suggestion>> The User Datagram Protocol (UDP) is **an unreliable and connectionless** protocol.

Reply:
@Amrita42 , I avoided "unreliable" because it feels like a judgement and makes UDP sound like a protocol users should not use. However, in streaming, VoIP, and other use cases, UDP is the better choice because it is OK if some packets get lost.
WDYT?

Reply:
@sbmetz , yes. I changed the sentence to your suggestion.

Reply:
Yes from that perspective , makes sense , please ignore this suggestion :)